qt/RpcQueue.h

```diff
@@ -65,7 +65,7 @@ private slots:
 
     // normal closure, takes response and returns new future
     template<typename Func, typename std::enable_if<
-        std::is_same<typename std::result_of<Func(RpcResponse const&)>::type, RpcResponseFuture>::value
+        std::is_same<typename std::invoke_result<Func, RpcResponse const&>::type, RpcResponseFuture>::value
```

Comment:
Here and down below:
```suggestion
        std::is_same_v<typename std::invoke_result_t<Func, RpcResponse const&>, RpcResponseFuture>
```

Reply:
Added 7dec44274fdb7bd5b834c7bb519a0b0c418ebdd1